service/src/main/java/bio/terra/workspace/service/resource/referenced/flight/create/CreateReferenceResourceFlight.java

```diff
@@ -21,10 +21,7 @@ public CreateReferenceResourceFlight(FlightMap inputParameters, Object beanBag)
 
     FlightBeanBag appContext = FlightBeanBag.getFromObject(beanBag);
 
-    // Perform access verification
-    addStep(new ValidateReferenceStep(appContext), RetryRules.cloud());
-
-    // If all is well, then store the reference metadata
+    // Store the reference metadata
```

Comment:
Apparently we didn't have any integration tests asserting that creation failed when there was no access. 🙃